spec/special-cases.js

```diff
@@ -9,7 +9,7 @@ describe("special cases", () => {
     const Parent = () => <div><NullComponent /></div>;
 
     return render(<Parent />).includeDataReactAttrs(false).toPromise().then(html => {
-      expect(html).to.equal("<div></div>");
+      expect(html).to.equal("<div><!-- react-empty --></div>");
```

Comment:
Shouldn't there be two tests here?

One that would be with `includeDataReactAttrs(false)` and that should return `"<div></div>"` and one that would be with `includeDataReactAttrs(true)` and that should return `"<div data-reactroot="" data-reactid="1"><!-- react-text: 1 --></div>"`